apps/docs/content/guides/platform/cost-control.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: 'cost-control'
+title: 'How to control your costs'
+---
+
+## Spend Cap
+The Spend Cap determines whether your organization can exceed the subscription plan's free quota allowance for any metric. Scenarios that could lead to high usage—and thus high costs—include system attacks or bugs in your software. The Spend Cap can protect you from these unexpected costs for certain metrics.
+
+### What happens when the Spend Cap is on?
+After exceeding the free quota allowance for a metric, further usage of that metric is disallowed until the next billing cycle. You don't get charged for over-usage but your services will be restricted according to our [Fair Use Policy](/docs/guides/platform/billing-faq#fair-use-policy) if you consistently exceed the free quota allowance.
+
+<Admonition type="note">
+  Note that only certain metrics are covered by the Spend Cap. Metrics with predictable costs are excluded.
+</Admonition>
+
+### What happens when the Spend Cap is off?
+Your projects will continue to operate after exceeding the free quota allowance for a metric. Any additional usage will be charged based on the metric's cost per unit, as outlined on the [pricing page](https://supabase.com/pricing).
+
+<Admonition type="note">
+  When the Spend Cap is off, we recommend monitoring your usage and costs on the [organization's usage page](https://supabase.com/dashboard/org/_/usage).
+</Admonition>
+
+### Metrics covered by the Spend Cap
+- Disk Size
+- Egress
+- Function Invocations
+- Monthly Active Users
+- Monthly Active SSO Users
+- Monthly Active Third Party Users
+- Realtime Messages
+- Realtime Peak Connections
+- Storage Image Transformations
+- Storage Size
+
+### Metrics not covered by the Spend Cap
```

Comment:
We might wanna add a sentence as to *why* these are not covered: They have predictable costs and are user choices